backend/__init__.py

```diff
@@ -0,0 +1,3 @@
+from backend.library.SimpleWebSocketServer import *
+
+name="backend"
```

Comment:
This whole frontend/backend naming convention is confusing. Backend doesn't equal server and frontend doesn't equal client.